test/translation/transformation/topLevelVariables.ts

```diff
@@ -0,0 +1,10 @@
+const obj = { value1: 1, value2: 2 };
```

Comment:
So do we want to continue using translation tests?

Reply:
I think in a small number of cases they are useful, especially now we use snapshot testing for them.